rules/false_positives/melange.yara

```diff
@@ -0,0 +1,14 @@
+rule melange_binary : override {
+  meta:
+    curl_tor_chmod_relative_run = "high"
+    description = "melange"
+    downgrade = "true"
```

Comment:
this field seems unnecessary

Reply:
Ah, yep. That's from an earlier version of the rules.

Reply:
Addressed in [`e716ba1` (#502)](https://github.com/chainguard-dev/malcontent/pull/502/commits/e716ba11a75e6cbd2e99b0f54f0d82f3deb46bef).